Views/TweetCell.h

```diff
@@ -17,11 +17,11 @@ NS_ASSUME_NONNULL_BEGIN
 @property (weak, nonatomic) IBOutlet UILabel *usernameHandleLabel;
 @property (weak, nonatomic) IBOutlet UILabel *timestampLabel;
 @property (weak, nonatomic) IBOutlet UILabel *tweetTextLabel;
-@property (weak, nonatomic) IBOutlet UIImageView *retweetIconView;
-@property (weak, nonatomic) IBOutlet UIImageView *favoriteIconView;
+@property (weak, nonatomic) IBOutlet UIButton *favoriteIconView;
```

Comment:
This is an argument against the naming convention used by a lot of iOS devs. The compiler won't complain that a `UIButton` is named `somethingView` but a reader of the code will be mislead.